packages/documentation/src/stories/components/alert.stories.tsx

```diff
@@ -0,0 +1,392 @@
+      }
+    },
+    show: {
+      name: 'Show',
+      control: { type: 'boolean' },
+      table: { disable: true }
+    }
+  }
+} as Meta;
+
+function onShowToggle (e: React.MouseEvent, args: Args, updateArgs: Function) {
+  e.preventDefault();
+  updateArgs({ show: !args.show });
+}
+
+const Template = (args: Args) => {
+  const [_, updateArgs] = useArgs();
+
+  const classes = [
+    'alert',
+    args.variant,
+    args.icon !== 'null' ? `pi-${args.icon}` : '',
+    args.noIcon ? 'no-icon' : '',
+    args.dismissible ? 'alert-dismissible' : '',
+    args.fixed ? 'alert-fixed-bottom' : '',
+    args.action ? 'alert-action' : '',
+    args.show ? '' : 'd-none'
+  ].filter(c => c).join(' ');
+
+  // every time you want to parse one or more elements to a JSX.Element[], concat all the elements to a string and parse them all
+  // even if it would be possible to write some of them directly
+  // the reason for this is, that react elements in a list need a "key" property, single elements don't
+  // and the parser only creates this property, when it receives multiple elements to parse
+  const content: JSX.Element[] | JSX.Element | string = parse(`<h4 class="alert-heading" key="title">${args.title}</h4>${args.content}`);
+
+  return <div className={ classes } role="alert">
+    {/* Dismissible Button */}
+    { (args.dismissible || args.fixed) && !args.action ? <button className="btn-close" data-dismiss="alert" aria-label="Close" onClick={ (e: React.MouseEvent) => onShowToggle(e, args, updateArgs) }></button> : null }
+
+    {/* Alert Content */}
+    { args.action ? <div className="alert-content">{ content }</div> : content }
+
+    {/* Alert Action Buttons */}
+    {
+      args.action ?
+      <div className="alert-buttons">
+        <button className="btn btn-primary btn-animated" onClick={ (e: React.MouseEvent) => onShowToggle(e, args, updateArgs) }><span>Akcepti</span></button>
+        <button className="btn btn-secondary btn-animated" onClick={ (e: React.MouseEvent) => onShowToggle(e, args, updateArgs) }><span>Aborti</span></button>
+      </div>
+      : null
+    }
+  </div>;
+};
+
+export const Default: Story = Template.bind({});
+Default.decorators = [
+  (Story: Story, { args }) => {
+    const [_, updateArgs] = useArgs();
+    const showToggleButton = args.fixed || args.action;
+    const showResetButton = !showToggleButton && args.dismissible && !args.show;
+
+    return <div>
+      { showToggleButton ? <button className="btn btn-secondary" onClick={ (e: React.MouseEvent) => onShowToggle(e, args, updateArgs) }>Toggle alert</button> : null }
+      { showResetButton ? <a href="#" onClick={ (e: React.MouseEvent) => onShowToggle(e, args, updateArgs) }>Show alert</a> : null }
+      <Story/>
+    </div>
+  }
+];
+
+export const Variant: Story = Template.bind({});
+Variant.parameters = {
+  controls: {
+    exclude: [
+      'Icon',
+      'Icon Color',
+      'No Icon',
+      'Dismissible',
+      'Fixed',
+      'Action',
```

Comment:
```suggestion
      'Action Buttons',
```

Most stories have this error, it needs to be replaced in each one.